Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/converter/ProcessConverter.java

```diff
@@ -84,18 +84,21 @@ private static Task getLastProcessedTask(Process process) {
      * <p>For process that have children, their repective tasks are also included.</p>
      * 
      * @param process the process
+     * @param considerChildren whether to include tasks of children processes
      * @return the list of tasks of the process (and potentially its children)
      */
-    private static List<Task> getListOfTasksForProgressCalculation(Process process) {
+    private static List<Task> getListOfTasksForProgressCalculation(Process process, Boolean considerChildren) {
```

Comment:
Why are you using nullable `Boolean` class instead of primitive `boolean` as the values should only be `true` or `false` ?

Reply:
Hi Henning, I didn't really think about it. Fixed it below.